src/components/alert/useAlertMessage.jsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react'
+
+export const useAlertMessage = () => {
+
+   const [alertMessage, setAlertMessage] = useState('')
```

Comment:
since alertMessage is a variable that is going to live inside of this custom hook, we can use a shorter name such as: message